qdrant-landing/themes/qdrant-2024/assets/css/partials/_qdrant-blog-post.scss

```diff
@@ -0,0 +1,166 @@
+@use '../helpers/functions' as *;
+
+.qdrant-blog-post {
+  &__header {
+    padding-top: $spacer * 2.5;
+    padding-bottom: $spacer * 1.5;
+    background-color: $neutral-98;
+    text-align: center;
+  }
+
+  &__title {
+    font-size: $spacer * 2;
+    line-height: rem(38);
+    margin-bottom: $spacer;
+    color: $neutral-20;
+  }
+
+  &__about {
+    display: flex;
+    justify-content: center;
+    gap: rem(6);
+    font-size: rem(14);
+    line-height: rem(21);
+    color: $neutral-50;
+    margin-bottom: $spacer * 2.5;
+
+    p {
+      margin-bottom: 0;
+    }
+  }
+
+  &__preview {
+    width: 100%;
+  }
+
+  &__body {
+    display: flex;
+    flex-direction: column;
+    gap: $spacer * 2.5;
+    padding-bottom: $spacer * 3;
+  }
+
+  &__social {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+
+    &-title {
+      font-size: rem(18);
+      line-height: rem(27);
+      font-weight: bold;
+      margin-bottom: 0;
+      padding-bottom: $spacer * 0.5;
+      border-bottom: rem(1) solid $primary-50;
+    }
+
+    &-links {
+      display: flex;
+      justify-content: flex-end;
+      align-items: center;
+      gap: $spacer * 2;
+    }
+  }
+
+  &__content {
```

Comment:
Could you please add an underline effect to the links on hover?

Reply:
done